docs/_content/standaard/catalogi/index.md

```diff
@@ -145,6 +145,17 @@ Bovendien gelden er beperkingen op verdere acties die uitgevoerd kunnen worden o
     * Er mag GEEN nieuw objecttype aangemaakt worden met een relatie naar een objecttype met `concept=false` (create)
     * Er mag GEEN nieuwe relatie worden gelegd tussen een objecttype en een objecttype met `concept=false` (update, partial_update)
 
+#### <a name="ztc-010">Relaties tussen objecten ([ztc-010](#ztc-010))</a>
+
+Het is NIET TOEGESTAAN dat objecten relaties hebben over verschillende catalogi
+heen. Zelfs als de catalogi hetzelfde zijn maar op verschillende endpoints 
+worden aangeboden mogen de relaties niet door elkaar gelegd worden.
+
+*Voorbeeld: Een `Zaaktype` in `Catalogus` X mag geen `Statustype` hebben uit
```

Comment:
Statustype heeft geen catalogus attribuut, dus ik denk dat `BesluitType` een beter voorbeeld is

Reply:
Denk dat dit voorbeeld prima werkt, het is iets specifieker omdat het op niveau van zaaktype zit ipv catalogus, maar het idee is hetzelfde.